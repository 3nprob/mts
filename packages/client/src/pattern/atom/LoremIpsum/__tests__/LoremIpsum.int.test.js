import {
  paragraphX2SmallTimesOneRender,
  paragraphX1SmallTimesOneRender,
  paragraphSmallTimesOneRender,
  paragraphMediumTimesOneRender,
  paragraphLargeTimesOneRender,
  textX2SmallTimesOneRender,
  textX1SmallTimesOneRender,
  textSmallTimesOneRender,
  textMediumTimesOneRender,
  textLargeTimesOneRender,
} from './LoremIpsum.int.render'

describe('<LoremIpsum />', () => {
  describe('Snaphot', () => {
    describe('Paragraph', () => {
      test('must match paragraphX2SmallTimesOneRender', () => {
        expect(global.renderToJSON(paragraphX2SmallTimesOneRender)).toMatchSnapshot()
      })

      test('must match paragraphX1SmallTimesOneRender', () => {
        expect(global.renderToJSON(paragraphX1SmallTimesOneRender)).toMatchSnapshot()
      })

      test('must match paragraphSmallTimesOneRender', () => {
        expect(global.renderToJSON(paragraphSmallTimesOneRender)).toMatchSnapshot()
      })

      test('must match paragraphMediumTimesOneRender', () => {
        expect(global.renderToJSON(paragraphMediumTimesOneRender)).toMatchSnapshot()
      })

      test('must match paragraphLargeTimesOneRender', () => {
        expect(global.renderToJSON(paragraphLargeTimesOneRender)).toMatchSnapshot()
      })
    })

    describe('Text', () => {
      test('must match textX2SmallTimesOneRender', () => {
        expect(global.renderToJSON(textX2SmallTimesOneRender)).toMatchSnapshot()
      })

      test('must match textX1SmallTimesOneRender', () => {
        expect(global.renderToJSON(textX1SmallTimesOneRender)).toMatchSnapshot()
      })

      test('must match textSmallTimesOneRender', () => {
        expect(global.renderToJSON(textSmallTimesOneRender)).toMatchSnapshot()
      })

      test('must match textMediumTimesOneRender', () => {
        expect(global.renderToJSON(textMediumTimesOneRender)).toMatchSnapshot()
      })

      test('must match textLargeTimesOneRender', () => {
        expect(global.renderToJSON(textLargeTimesOneRender)).toMatchSnapshot()
      })
    })
  })
})
