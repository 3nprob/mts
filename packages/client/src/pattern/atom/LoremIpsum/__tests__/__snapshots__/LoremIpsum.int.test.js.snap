// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoremIpsum /> Snaphot Paragraph must match paragraphLargeTimesOneRender 1`] = `
<p
  className=""
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie. Morbi ac dapibus urna. Donec magna felis, porta eget felis vel, sodales fringilla quam. Maecenas mauris ligula, tempus et pretium quis, aliquam nec odio. Praesent mattis a leo lacinia facilisis. Integer rutrum ac purus nec blandit. Nullam et felis ut eros sodales laoreet non vel ex. Suspendisse quis viverra purus. Pellentesque lobortis semper laoreet. Proin venenatis lacus nec neque aliquam cursus. Nullam porttitor a nulla a venenatis. Duis eu sapien velit. Phasellus sagittis vitae ligula eget posuere. Nullam volutpat cursus nunc, at molestie tellus pellentesque non. Vestibulum dignissim ante vel diam tincidunt iaculis. Vestibulum vehicula lobortis ultrices.
</p>
`;

exports[`<LoremIpsum /> Snaphot Paragraph must match paragraphMediumTimesOneRender 1`] = `
<p
  className=""
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie.
</p>
`;

exports[`<LoremIpsum /> Snaphot Paragraph must match paragraphSmallTimesOneRender 1`] = `
<p
  className=""
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
</p>
`;

exports[`<LoremIpsum /> Snaphot Paragraph must match paragraphX1SmallTimesOneRender 1`] = `
<p
  className=""
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
</p>
`;

exports[`<LoremIpsum /> Snaphot Paragraph must match paragraphX2SmallTimesOneRender 1`] = `
<p
  className=""
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien.
</p>
`;

exports[`<LoremIpsum /> Snaphot Text must match textLargeTimesOneRender 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie. Morbi ac dapibus urna. Donec magna felis, porta eget felis vel, sodales fringilla quam. Maecenas mauris ligula, tempus et pretium quis, aliquam nec odio. Praesent mattis a leo lacinia facilisis. Integer rutrum ac purus nec blandit. Nullam et felis ut eros sodales laoreet non vel ex. Suspendisse quis viverra purus. Pellentesque lobortis semper laoreet. Proin venenatis lacus nec neque aliquam cursus. Nullam porttitor a nulla a venenatis. Duis eu sapien velit. Phasellus sagittis vitae ligula eget posuere. Nullam volutpat cursus nunc, at molestie tellus pellentesque non. Vestibulum dignissim ante vel diam tincidunt iaculis. Vestibulum vehicula lobortis ultrices."`;

exports[`<LoremIpsum /> Snaphot Text must match textMediumTimesOneRender 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie."`;

exports[`<LoremIpsum /> Snaphot Text must match textSmallTimesOneRender 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt."`;

exports[`<LoremIpsum /> Snaphot Text must match textX1SmallTimesOneRender 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante."`;

exports[`<LoremIpsum /> Snaphot Text must match textX2SmallTimesOneRender 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien."`;
